pup-0004.md

```diff
@@ -0,0 +1,115 @@
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+### Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+### Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at pulp-conduct@redhat.com. All
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+### Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+[homepage]: https://www.contributor-covenant.org
+
+## Displaying the CoC
+
+A link to the CoC above will be provided in the "Code of Conduct" section of
+CONTRIBUTING.md file in the root of the pulp Git repository.
+
+## Drawbacks
+
+This CoC does not identify anything specific about the Pulp community.
+
+## Alternatives
+
+### Not adopting a CoC
+
+We can continue to operate without a Code of Conduct and hope that we can
+foster an open and inclusive environment without it.
+
+### Adopt a CoC similar to Django's
+
+The [Django CoC](https://www.djangoproject.com/conduct/) is an alternative code
+of conduct that we could adopt.
+
+### Adop a CoC that we write as a committee
```

Comment:
s/Adop/Adopt/

Reply:
I think this could say something like:  We probably could not produce a better document since we have no background in this area.